src/pages/SeriesDetailsPage.js

```diff
@@ -15,7 +15,7 @@ export default function SeriesDetailsPage({
 }) {
   const { id } = useParams();
   const navigate = useNavigate();
-  const { data: seriesDetails, isLoading } = useSeriesDetails(id);
+  const { data: seriesDetails, isLoading } = useSeriesDetails({ id });
```

Comment:
Why are you passing the id as object? 🤔

Reply:
This was only a test for the backend. I am not sure yet if I need this as an object or not. That will show in the next US 🙂